pycbc/io/hdf.py

```diff
@@ -641,12 +641,11 @@ def newsnr_sgveto_psdvar(self):
                                            self.sgchisq, self.psd_var_val)
 
     def get_column(self, cname):
-        if self.mask is not None:
+        if self.mask is not None and (self.mask.sum() != len(self.mask)):
```

Comment:
It might be worth doing things the old way if `self.mask.sum()` is at least 30% of the full set (say)

Reply:
Yeah, that would make sense.